plugins/clamd.js

```diff
@@ -213,15 +243,23 @@ exports.hook_data_post = function (next, connection) {
                 txn.results.add(plugin, {pass: 'clean', emit: true});
                 return next();
             }
+
             var m = /^stream: (\S+) FOUND/.exec(result);
             if (m) {
                 var virus;                                   // Virus found
                 if (m[1]) { virus = m[1]; }
                 txn.results.add(plugin, {
-                    fail: 'virus' + (virus ? ('(' + virus + ')') : ''),
+                    fail: virus ? virus : 'virus',
                     emit: true
                 });
 
+                if (virus && plugin.rejectRE &&       // enabled
```

Comment:
What bothers me about this is:

``````
 var rejectOpts = [
       'Broken.Executable', 'Structured', 'Encrypted', 'PUA',
       'OLE2', 'Safebrowsing', 'UNOFFICIAL', 'Phishing',
  ];
`````

Will create regexps which are not anchored and have a potential to mis-fire where it shouldn't e.g. if you load and want to reject on PhishTank data (e.g. an UNOFFICIAL signature), then this regexp will incorrectly hit it.   ClamAV recommends the Phishing signatures should be scored; but it means Phishing *heuristics* should be scored.

IMO - the regexps should be anchored in some way and have more surrounding context e.g.:

`````
Broken.Executable =>  ^Broken\.Executable.?
Structured => ^Heuristics\.Structured\.
Encrypted => ^Encrypted\.
PUA => ^PUA\.
OLE2 => ^Heuristics\.OLE2\.ContainsMacros
Safebrowsing => ^Heuristics\.Safebrowsing\.
Phishing => ^Heuristics\.Phishing\.
UNOFFICIAL => \.UNOFFICIAL$
`````